share/p2p/shrexnd/server.go

```diff
@@ -130,6 +131,7 @@ func (srv *Server) handleNamespacedData(ctx context.Context, stream network.Stre
 
 	shares, err := srv.getter.GetSharesByNamespace(ctx, dah, req.NamespaceId)
 	if errors.Is(err, share.ErrNotFound) {
+		logger.Warn("server: nd not found")
```

Comment:
```suggestion
		logger.Warn("server-nd: shares not found")
```

pls also log dah hash and namespaceID 

This could be a frequent log tho which might trip out users but 🤷🏻

Reply:
server logs namespace id, it is inside logger